src/core/control/Control.cpp

```diff
@@ -1839,6 +1839,9 @@ void Control::toolChanged() {
         toolColorChanged();
     }
 
+    if (toolHandler->getToolType() == TOOL_PEN)
+        toolLineStyleChanged();
```

Comment:
Please always encapsulate conditional code with braces (see [our coding conventions](https://github.com/xournalpp/xournalpp/wiki/Coding-conventions#indentation))
```suggestion
    if (toolHandler->getToolType() == TOOL_PEN) {
        toolLineStyleChanged();
    }
```

Reply:
Fixed by 923338c2.